hpcbench/driver.py

```diff
@@ -808,17 +923,16 @@ def srun(self):
 
         :rtype: string
         """
-        srun = self.campaign.process.config.get('srun') or 'srun'
-        return find_executable(srun)
+        commands = self.campaign.process.get('commands', {})
+        return find_executable(commands.get('srun', 'srun'))
 
     @cached_property
     def common_srun_options(self):
         """Get options to be given to all srun commands
 
         :rtype: list of string
         """
-        slurm_config = self.campaign.process.get('config', {})
-        return slurm_config.get('srun_options') or []
+        return self.campaign.process.get('srun', {})
```

Comment:
If a user writes such yaml, `self.campaign.process.get('srun', {})` will be equal to `None`
```yaml
process:
  srun:
```

This is why I prefer using `self.campaign.process.get('srun') or {}`